### DISCUSSION TITLE
South Africa is home to the "Big Five" game animals, what are they?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#654] [Galaxy Trivia] [answered]
*list them all as emojis*

**Comment by DevElkami** [comment]
Lion :tiger2:, leopard :leopard:, rhino :ox:, elephant :elephant: and African buffalo :water_buffalo:.

**Comment by FleetAdmiralJakob** [comment, marked as answer]
🦁 Lion

🐘 Elephant

🦏 Rhino

🐆 Leopard

🐃 Cape Buffalo
